src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -199,6 +201,20 @@ public final class Guardrails implements GuardrailsMBean
                    state -> CONFIG_PROVIDER.getOrCreate(state).getCompactTablesEnabled(),
                    "Creation of new COMPACT STORAGE tables");
 
+    /**
+     * Guardrail disabling the creation of a new table when {@link Option#DEFAULT_TIME_TO_LIVE} is set to 0
+     * and compaction strategy is {@link TimeWindowCompactionStrategy} or its subclass. If this guardrail does not fail
+     * and such configuration combination is found, it will always warn.
+     */
+    public static final EnableFlag zeroDefaultTTLOnTWCSEnabled =
+    new EnableFlag("zero_default_ttl_on_time_window_compaction_strategy_enabled",
```

Comment:
This should be the name of the guardrail, not the name of the property.  So either`zero_default_ttl_on_time_window_compaction_strategy` or `zero_default_ttl_on_twcs`